cmd/kubelet/app/options/options.go

```diff
@@ -416,8 +416,8 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 
 	fs.Int32Var(&c.RegistryPullQPS, "registry-qps", c.RegistryPullQPS, "If > 0, limit registry pull QPS to this value.  If 0, unlimited.")
 	fs.Int32Var(&c.RegistryBurst, "registry-burst", c.RegistryBurst, "Maximum size of a bursty pulls, temporarily allows pulls to burst to this number, while still not exceeding registry-qps. Only used if --registry-qps > 0")
-	fs.Int32Var(&c.EventRecordQPS, "event-qps", c.EventRecordQPS, "QPS to limit event creations. The number must be >= 0. If 0 will use DefaultQPS: 5.")
-	fs.Int32Var(&c.EventBurst, "event-burst", c.EventBurst, "Maximum size of a bursty event records, temporarily allows event records to burst to this number, while still not exceeding event-qps. The number must be >= 0. If 0 will use DefaultBurst: 10.")
+	fs.Int32Var(&c.EventRecordQPS, "event-qps", c.EventRecordQPS, "QPS to limit event creations. The number must be >= 0. If 0 will use DefaultQPS: 50.")
+	fs.Int32Var(&c.EventBurst, "event-burst", c.EventBurst, "Maximum size of a bursty event records, temporarily allows event records to burst to this number, while still not exceeding event-qps. The number must be >= 0. If 0 will use DefaultBurst: 100.")
```

Comment:
Shouldn't this read:

"The number must be >= 0. The default value is 100. If 0 will use DefaultBurst: 10." (since DefaultBurst is defined in client-go and EventBurst's default value is defined in KubeletConfiguration)

Reply:
also reference to the value in config file needs to be made. So it is clear that the value may be also set in other place

Reply:
@SergeyKanzhelev what exactly are you asking for? IIUC, the CLI flags will override any config value set in the config file. Do we want to mention that here?

Reply:
For the definition of default values you can refer:
https://github.com/kubernetes/kubernetes/blob/20fc8a939c3394bd31edd06a893b75d1c312e27c/pkg/kubelet/apis/config/v1beta1/defaults.go#L98-L103